recipes/jsoncpp/all/test_package/CMakeLists.txt

```diff
@@ -1,11 +1,16 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.1)
 project(test_package)
 
 SET(CMAKE_CXX_STANDARD 11)
 SET(CMAKE_CXX_STANDARD_REQUIRED ON)
 SET(CMAKE_CXX_EXTENSIONS OFF)
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+
+find_package(JsonCpp REQUIRED)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
-target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+target_link_libraries(${PROJECT_NAME} JsonCpp::JsonCpp)
+
+if(JsonCpp_VERSION VERSION_LESS "1.0.0")
+    # For versions 0.y.z, includes must be prefixed with "jsoncpp/"
+    target_compile_definitions(${PROJECT_NAME} PRIVATE WITH_JSONCPP_PREFIX)
```

Comment:
you have to pass `WITH_JSONCPP_PREFIX` by `self.cpp_info.defines`

Reply:
It's not from upstream. This is only for the test_package since the major versions have different header include.

Reply:
Thanks for clarifying!